test/pkcs12_api_test.c

```diff
@@ -35,7 +35,7 @@ static PKCS12 *PKCS12_load(const char *fpath)
     BIO *bio = NULL;
     PKCS12 *p12 = NULL;
 
-    bio = BIO_new_file(fpath, "r");
+    bio = BIO_new_file(fpath, "rb");
```

Comment:
Are we sure this won't break platforms that do not know the "b" flag? Perhaps better to #ifdef __WIN32 it?

Reply:
If it really breaks, then some other tests need modification:

https://github.com/openssl/openssl/blob/fc2be2d07acc0cfe954320c2491b8c5461cbef09/test/sslapitest.c#L1375
https://github.com/openssl/openssl/blob/fc2be2d07acc0cfe954320c2491b8c5461cbef09/test/evp_pkey_provided_test.c#L101

Reply:
OK then.